<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>characteristic</class>
 <widget class="QDialog" name="characteristic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Characteristic</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <property name="verticalSpacing">
    <number>12</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QLabel" name="_nameLit">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>_name</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="XLineEdit" name="_name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="_typeLit">
       <property name="text">
        <string>Type:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="XComboBox" name="_type">
         <item>
          <property name="text">
           <string>Text</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>List</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Date</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_orderLit">
         <property name="text">
          <string>Order:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="_order"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="_search">
       <property name="text">
        <string>Include in searches</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QDialogButtonBox" name="_buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="_useGroup">
     <property name="title">
      <string>May be used on:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QCheckBox" name="_quotes">
        <property name="text">
         <string>Quote</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="_items">
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="_contacts">
        <property name="text">
         <string>Contact</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="_incidents">
        <property name="text">
         <string>Incident</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="_opportunity">
        <property name="text">
         <string>Opportunity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="_employees">
        <property name="text">
         <string>Employee</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="_addresses">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="_salesorders">
        <property name="text">
         <string>Sales Order</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="_invoices">
        <property name="text">
         <string>Invoice</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="_lotSerial">
        <property name="text">
         <string>Lot/Serial</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="_customers">
        <property name="text">
         <string>Customer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="_crmaccounts">
        <property name="text">
         <string>Account</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="_vendors">
        <property name="text">
         <string>Vendor</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="_purchaseorders">
        <property name="text">
         <string>Purchase Order</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QCheckBox" name="_vouchers">
        <property name="text">
         <string>Voucher</string>
        </property>
       </widget>
      </item>     
      <item row="6" column="0">
       <widget class="QCheckBox" name="_projects">
        <property name="text">
         <string>Projects</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="_tasks">
        <property name="text">
         <string>Project Tasks</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTabWidget" name="_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="_optionsPage">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="_stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="_textPage">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="_maskLit">
                   <property name="text">
                    <string>Input Mask:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="XComboBox" name="_mask">
                   <property name="editable">
                    <bool>true</bool>
                   </property>
                   <property name="allowNull">
                    <bool>true</bool>
                   </property>
                   <item>
                    <property name="text">
                     <string>00/00/0000</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>###-###-####</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>###-##-####</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>00,000.00</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="_validatorLit">
                   <property name="text">
                    <string>Validator:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="XComboBox" name="_validator">
                   <property name="editable">
                    <bool>true</bool>
                   </property>
                   <property name="allowNull">
                    <bool>true</bool>
                   </property>
                   <property name="nullStr">
                    <string/>
                   </property>
                   <item>
                    <property name="text">
                     <string notr="true">[Y|N]</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string notr="true">\S+</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string notr="true">[1-9]\d{0,3}</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string notr="true">[A-Z]\d{5}[1-9]</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="_listPage">
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="_new">
               <property name="text">
                <string>New</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="_delete">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QTreeView" name="_charoptView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="_datePage">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>No Options</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_descripPage">
      <attribute name="title">
       <string>Description</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="XTextEdit" name="_description"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XTextEdit</class>
   <extends>QTextEdit</extends>
   <header>xtextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_name</tabstop>
  <tabstop>_type</tabstop>
  <tabstop>_order</tabstop>
  <tabstop>_addresses</tabstop>
  <tabstop>_contacts</tabstop>
  <tabstop>_customers</tabstop>
  <tabstop>_crmaccounts</tabstop>
  <tabstop>_employees</tabstop>
  <tabstop>_incidents</tabstop>
  <tabstop>_items</tabstop>
  <tabstop>_lotSerial</tabstop>
  <tabstop>_opportunity</tabstop>
  <tabstop>_tab</tabstop>
  <tabstop>_mask</tabstop>
  <tabstop>_validator</tabstop>
  <tabstop>_charoptView</tabstop>
  <tabstop>_new</tabstop>
  <tabstop>_delete</tabstop>
  <tabstop>_description</tabstop>
  <tabstop>_buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>characteristic</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_type</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>_stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
